### PR TITLE
[ET-910] Fix failing tests and view link template problem

### PR DESCRIPTION
**Pull request by sc0ttkclark**
[ET-910]

Fix failing tests from G20.09 and G20.08 work and the view links template errors in testing suite.

Screenshot
image

[ET-910]: https://moderntribe.atlassian.net/browse/ET-910

### CODE REVIEW

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 6 error(s)
:warning: 2 warning(s)

File: src/views/tickets/view-link.php [hunk 1]
```diff
@@ -55,13 +56,13 @@
 	return false;
 }
 
-$link = $view->get_tickets_page_url( $event_id, $is_event_page );
+$link = $view->get_tickets_page_url( $post_id, $is_event_page );
```
Comment (tr1b0t) [inline src/views/tickets/view-link.php:59]: :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $link

<sub>hash: 064214d8d15658111af8ebae4fadea21</sub>

File: src/views/tickets/view-link.php [hunk 2]
```diff
@@ -55,13 +56,13 @@
...
 
 // Translators: 1: number of RSVPs and/or Tickets with accompanying ticket type text, 2: post type label.
 $message = esc_html( sprintf( __( 'You have %1s for this %2s.', 'event-tickets' ), implode( _x( ' and ', 'separator if there are both RSVPs and Tickets', 'event-tickets' ), $counters ), $post_type_singular ) );
 ?>
 
 <div class="tribe-link-view-attendee">
 	<?php echo $message ?>
-	<a href="<?php echo esc_url( $link ) ?>"><?php echo sprintf( esc_html__( 'View your %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, $user_id ) ) ?></a>
-</div>
\ No newline at end of file
```
Comment (tr1b0t) [inline src/views/tickets/view-link.php:67]: * :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$view'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$user_id'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$user_id'.

<sub>hash: 1990b441e8d126d73a3b47cf095c345e</sub>

File: src/views/tickets/view-link.php [hunk 3]
```diff
@@ -12,6 +12,7 @@
  * @since   4.10.9  Use customizable ticket name functions.
  * @since   4.11.0 Made template more like new blocks-based template in terms of logic.
  * @since 4.12.1 Account for empty post type object, such as if post type got disabled. Fix typo in sprintf placeholders.
+ * @since TBD Add additional checks to prevent PHP errors when called from automated testing.
  *
  * @version 4.12.1
```
Comment (sc0ttkclark) [inline src/views/tickets/view-link.php:17]:
```suggestion
 * @version TBD
```

**Review by juanfra — APPROVED** [review]
🚀

**Pull request merged**